app/lib/service/email/backend.dart

```diff
@@ -118,6 +123,10 @@ class EmailBackend {
         sent.add(recipientEmail);
       } catch (e, st) {
         _logger.warning('Email sending failed (claimId="$claimId").', e, st);
+        if (emailSender.shouldBackoff) {
+          resetAttempt = true;
+          break;
+        }
```

Comment:
I don't understand this.

Are we using `emailSender.shouldBackoff` as a return value?

That we should backoff is not an indicator that it's reasonable to try sending the email again, is it?

Reply:
`emailSender` hides the `SmtpClientAuthenticationException` and returns an `EmailSenderException` instead. This was a quick hack to track the exception type, but I shall rather extend the `EmailSenderException` to indicate if the attempt should be reset.

Reply:
Using the status code >= 500 now...